games-roguelike/FTL-gog/FTL-gog-1.6.3.ebuild

```diff
@@ -0,0 +1,57 @@
+# Copyright 1999-2018 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+
+inherit desktop
+
+DESCRIPTION="Top-down roguelike space ship simulator"
+HOMEPAGE="https://www.gog.com/game/faster_than_light"
+SRC_URI="ftl_advanced_edition_en_1_6_3_17917.sh"
+RESTRICT="bindist fetch strip test"
+
+LICENSE="all-rights-reserved"
+SLOT="0"
+KEYWORDS="-* ~amd64 ~x86"
+IUSE=""
+
+DEPEND="app-arch/unzip"
+RDEPEND="media-libs/freetype
+	media-libs/libsdl[X,sound,opengl,video]
+	media-libs/libpng
+	sys-libs/zlib
+	virtual/opengl"
+
+S="${WORKDIR}"
+
+pkg_nofetch() {
+	elog "Please download the installer from GOG to:"
+	elog "   ${DISTDIR}"
+}
+
+src_unpack() {
+	# Strip the installer shell script from the binary, then unzip
+	dd if="${DISTDIR}/${A}" bs=680071 skip=1 of=ftl.zip
+	unpack "${WORKDIR}/ftl.zip"
+}
+
+src_prepare() {
+	# Discard useless installer and Ubuntu-specific files from GOG
+	rm -r data/noarch/support/{gog*.sh,xdg-utils}
+	default
+}
+
+src_install() {
+	insinto /opt/gog/FTL
+	doins -r data/noarch/*
```

Comment:
I don't have this installer to check, but at least in Baldur's Gate, I just installed `data/noarch/game` into `/opt`. The manuals go into the docdir, and the support stuff was not useful.

If you did that, you could drop your `src_prepare`.